src/Compilers/Core/Portable/Operations/IMultiVariableDeclaration.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents multiple declared variables in a single declarator.
+    /// </summary>
+    /// <para>
+    /// Current Usage:
+    ///   (1) VB As New statements
```

Comment:
>VB As New statements [](start = 14, length = 20)

How is this different from the second case? BTW, is there a such thing as "As New statement"?

Reply:
Simplified these to VB Dim statement groups and VB Using statement groups.

---
In reply to: [146930446](https://github.com/dotnet/roslyn/pull/22842#discussion_r146930446) [](ancestors = 146930446)